airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -322,5 +322,5 @@ def _set_resources(resources):
     def _set_name(self, name):
         if self.pod_template_file or self.full_pod_spec:
             return None
-        validate_key(name, max_length=63)
+        validate_key(name, max_length=220)
```

Comment:
LGTM, we can keep it on 220 also for v1.10, as preparation for the longer suffix added in v2.0

Reply:
Any further action needed from my side? :thinking:

Reply:
I'm not sure what else is needed for this fix to get on the v1.10 merge train :steam_locomotive:

merging is blocked as long as there are unticked TODOs in the PR description btw.

maybe @kaxil: anything else?

Reply:
Can we add tests too, please?

@dimberman Can you take a look please at this PR?

Reply:
LGTM just would like to see some tests.